docs/docs/messaging-and-voice-channels.mdx

```diff
@@ -12,7 +12,7 @@ in a `credentials.yml` file.
 An example file is created when you run `rasa init`, so it's easiest to edit that file
 and add your credentials there. Here is an example with Facebook credentials:
 
-```yaml
+```yaml title="credentials.yml"
```

Comment:
Very cool, didn't know we could do this 😎

Reply:
yes it's a lot nicer than our usual: "... into your `xyz.yaml` put: "